students/etwum/lesson09/mailroom_part5.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+# initial list of donors
+
+
+class DonorData:
+    def __init__(self,donors):
+        self.donors = donors
+
+    def donor_data(self):
+        return self.donors
```

Comment:
would be good if your Donor class had `name` attribute as well as total, avg, donations etc basically everything directly related to the Donor object